tests/test_pds3file.py

```diff
@@ -127,6 +129,59 @@ def test_image_save_samefile():
         image.save(image.filename)
 
 
+def test_image_save_lines_linesamples():
+    image = PDS3Image.open(filename)
+    image.data = image.data[0, 3:8, 3:8]
+    image.save('Temp_Image.IMG')
+    image_temp = PDS3Image.open('Temp_Image.IMG')
+    assert image_temp.lines == 5
+    assert image_temp.samples == 5
+    os.remove('Temp_Image.IMG')
```

Comment:
You should probably use `try: ... finally: os.remove...` here to ensure that the `Temp_Image.IMG` gets removed in all cases.